crates/swc_ecma_parser/src/error.rs

```diff
@@ -83,7 +83,7 @@ pub enum SyntaxError {
     UnterminatedStrLit,
     ExpectedUnicodeEscape,
     EscapeInReservedWord {
-        word: JsWord,
```

Comment:
This is done so that, in addition to printing the word in an error message, the whole `Token` that caused the error can be 'put back into the parser'.